cmd/cmd_utils.go

```
@@ -605,3 +605,39 @@ func getConfigAndStacksInfo(commandName string, cmd *cobra.Command, args []strin
 }
+
+func stackFlagCompletion(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
+	output, err := listStacks(cmd)
+	if err != nil {
+		return nil, cobra.ShellCompDirectiveNoFileComp
+	}
+	return output, cobra.ShellCompDirectiveNoFileComp
+}
+
+func AddStackCompltion(cmd *cobra.Command) {
+	cmd.RegisterFlagCompletionFunc("stack", stackFlagCompletion)
```

golangci-lint (1.62.2)

Error return value of `cmd.RegisterFlagCompletionFunc` is not checked

(errcheck)